tests/test_visibilities.py

```diff
@@ -0,0 +1,14 @@
+"""Test suite for parameters module"""
+
+# pylint: disable=too-few-public-methods
+
+from luxtronik.visibilities import Visibilities
+
+
+class TestVisibilities:
+    """Test suite for Visibilities"""
+
+    def test_init(self):
+        """Test cases for initialization"""
+        visibilities = Visibilities()
+        assert visibilities.name == "Visibility"
```

Comment:
This test might be missing for `Parameter`, as I can only see it for `Visibility` and `Calculation`.

Reply:
Added.